source/style-guide.erb

```diff
@@ -123,6 +161,77 @@ title: "Style guide"
     </div>
   </div>
 
+  <div class="style-guide__container">
+    <div class="style-guide__container-title">
+      Product story large tile
+    </div>
+    <div class="style-guide__container-body--grey">
+      <div class="product-story-large-tile">
+        <div class="product-story-large-tile__container">
+          <%= image_tag "http://via.placeholder.com/616x244", alt: "product story client", class: "product-story-large-tile__image" %>
+          <div class="product-story-large-tile__content">
+            <div class="product-story-large-tile__client">Product story client name</div>
+            <div class="product-story-large-tile__title">Product story title</div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="style-guide__container">
+    <div class="style-guide__container-title">
+      Product story small tile
+    </div>
+    <div class="style-guide__container-body--grey">
+      <div class="product-story-small-tile">
+        <div class="product-story-small-tile__container">
+          <%= image_tag "http://via.placeholder.com/454x180", alt: "product story client", class: "product-story-small-tile__image" %>
+          <div class="product-story-small-tile__content">
+            <div class="product-story-small-tile__client">Product story client name</div>
+            <div class="product-story-small-tile__title">Product story title</div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="style-guide__container">
+    <div class="style-guide__container-title">
+      Person tile
+    </div>
+    <div class="style-guide__container-body">
+      <div class="person-tile">
+          <div class="person-tile__content">
+            <div class="anchor">
+              <span id="style-guide-person-tile"></span>
+            </div>
+            <div class="person-tile__image-container">
+              <img class="person-tile__image" src="/assets/images/people/tom-sabin.png" alt="tom sabin" />
```

Comment:
I wonder if we should also use a placeholder image with dimensions here too. I think it should help when people add new caricatures to use the correct size.